packages/convergence/CHANGELOG.md

```diff
@@ -13,11 +13,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 ### Changed
 
 - updated inline api docs
+- `Convergence` assertions retain the instance context
 
 ### Deprecated
 
 - `.once()` in favor of `.when()`
 
+### Removed
+
+- context currying from `convergeOn`
```

Comment:
Is this a breaking change of any sort? I'm trying to think of a way you'd use this while testing..

Reply:
It used to be used by `@bigtest/mocha` via `convergeOn.call(this, assertion)`. 

`convergeOn`, however, is meant to be internal. I think by v1 we should stop exporting it.

Reply:
Yea, I agree. We should open an issue to track that.

I bet we could get away with this breaking change. _oof famous last words_